spec/spec_helper.rb

```diff
@@ -5,9 +5,9 @@
 
 require 'simplecov'
 SimpleCov.formatter = Coveralls::SimpleCov::Formatter
-SimpleCov.start do
-  add_filter 'spec'
-end
+SimpleCov.add_filter 'spec'
+
+SimpleCov.start 'rails' unless SimpleCov.running
```

Comment:
but ... this gem isn't rails ...

See also https://github.com/sul-dlss/moab-versioning/pull/193.

I'm fine with this to get it going ...